CHANGELOG

```diff
@@ -1,6 +1,17 @@
 devel
 -----
 
+* APM-283: Use parallel gather in almost all queries. The only case where we
+  cannot use parallel gather is when using traversals, although there are some
+  exceptions for disjoint SmartGraphs where the traversal can run completely on
+  the local DB-server. All other queries should now be able to parallelize the
+  gather node. This can not only speed up queries quite significantly, but also
+  reduce the memory required by the coordinator for queries like
+  `FOR i IN @n INSERT {} INTO collection`. The serial gather node processes one
+  DB-server after the other, but since the new documents should be distributed
+  evenly over all shards, the coordinator has to buffer documents that need to
+  be sent to other servers.
```

Comment:
Thinking about this sentence about the "serial gather node", I think this could be a bit difficult to understand.
Because we making things _parallel_, and not _serial_. 
IMO w should instead write something like "this also overcomes issues with the previous serial processing within gather nodes, which could lead to high memory usage on coordinators for buffering the documents for all shards, and to timeouts on some DB-Servers because of query parts being idle for too long".
Just a suggestion. The exact wording can probably still be improved.